ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -371,14 +381,22 @@ internal static class ArchiKestrel {
 		IMvcBuilder mvc = services.AddControllers();
 
 		// Add support for controllers declared in custom plugins
-		if (PluginsCore.ActivePlugins.Count > 0) {
+		if (PluginsCore.ActivePluginsCount > 0) {
```

Comment:
And here :stuck_out_tongue:

Reply:
STOP! I'm doing search-and-replace anyways. You don't need to point out every single one of them.

Reply:
https://www.youtube.com/watch?v=HgzGwKwLmgM

Reply:
```
:%s/ActivePluginsCount/ActivePlugins.Count/g

```